--- a/src/OrchardCore/OrchardCore.SourceGenerators/ArgumentsFromInterceptor.cs
+++ b/src/OrchardCore/OrchardCore.SourceGenerators/ArgumentsFromInterceptor.cs
@@ -1,4 +1,5 @@
 using System.Collections.Immutable;
+using System.Security.Cryptography;
 using System.Text;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
@@ -168,7 +169,8 @@
 
     private static void GenerateInterceptor(StringBuilder sb, InvocationInfo info)
     {
-        var uniqueId = Guid.NewGuid().ToString("N");
+        using var sha256 = SHA256.Create();
+        var uniqueId = BitConverter.ToString(sha256.ComputeHash(Encoding.UTF8.GetBytes(info.Location.Data))).Replace("-", "");
 
         sb.AppendLine($"    file static class Interceptor_{uniqueId}");
         sb.AppendLine("    {");
